e2e/rbd_helper.go

```diff
@@ -314,6 +314,38 @@ func deleteBackingRBDImage(f *framework.Framework, pvc *v1.PersistentVolumeClaim
 	return err
 }
 
+func backingRBDImageHasExtents(f *framework.Framework, pvc *v1.PersistentVolumeClaim) (bool, error) {
```

Comment:
what are we validating here? just running `rbd diff` but not checking anything from the command output?

Reply:
`rbd diff` returns no output in case the image does not contain any data. There will be a line for each extent when something is written (allocated) there.

Reply:
Added comment to explain the need/usage of the function.